principal/options.go

```diff
@@ -377,3 +377,10 @@ func WithResourceProxyTLS(tlsConfig *tls.Config) ServerOption {
 		return nil
 	}
 }
+
+func WithWaitDurationForAgentPing(interval time.Duration) ServerOption {
```

Comment:
Similar comment as for the command line. This should be renamed to be more in line with what it does.